multichain/openrpc.yaml

```diff
@@ -398,7 +406,7 @@ components:
     SessionScopes:
       type: object
       patternProperties:
-        "[-a-z0-9]{3,8}:[-_a-zA-Z0-9]{1,32}":
+        "[-a-z0-9]{3,8}(:[-_a-zA-Z0-9]{1,32})?":
```

Comment:
changed this to match `ScopeString` regex. which allows references to be optional